creator-node/prod.env

```diff
@@ -31,7 +31,7 @@ timeout=3600000
 enforceWriteQuorum=false
 otelTracingEnabled=true
 otelCollectorUrl=https://opentelemetry-collector.audius.co/v1/traces
-maxUpdateReplicaSetJobConcurrency=10
+maxUpdateReplicaSetJobConcurrency=0
```

Comment:
just flagging that the code isn't made to have this value as 0 actually actually pause the queue. it will make each worker have 1 because of the cluster worker logic

Reply:
to make it pause in stateReconciliation/index.js it needs:
```js
if (config.get('maxUpdateReplicaSetJobConcurrency') === 0) {
      await queue.pause()
      return
}
```

Reply:
thank you - adding that code there, updated this PR tho